cmd.go

```diff
@@ -83,10 +83,23 @@ func (e *Executor) Execute() error {
 			return err
 		}
 		for name, repo := range cfg.Repos {
-			for _, cherry := range cfg.Patches[name] {
-				err = repo.Cherry(e.Dir(), name, cherry)
-				if err != nil {
-					return fmt.Errorf("Unable to apply %s to %s: "+cherry, name, err.Error())
+			if cfg.Patches[name] != nil {
+				for _, cherry := range cfg.Patches[name].Changesets {
+					err = repo.Cherry(e.Dir(), name, cherry)
+					if err != nil {
+						return fmt.Errorf("Unable to apply %s to %s: "+cherry, name, err.Error())
+					}
+				}
+				subscriptions := cfg.Patches[name].Subscriptions
```

Comment:
So, this means that all subscriptions are applied after all patches. This isn't necessarily always right. Sometimes you'll need individual patches before and after subscriptions.

Is it possible to roll it all into a single object like:

```    superwidget:
      changesets:
      - {changeset1}
      - branch: remote01/widget_feature
        changesets:
          - {changesetA}
          - {changesetB}
      - {changeset2}
```
?

Reply:
Updated to be backward compatible and subscriptions/changesets are now interchangeable.